app/src/main/java/ch/epfl/culturequest/backend/artprocessing/apis/WikipediaDescriptionApi.java

```diff
@@ -51,8 +57,11 @@ public CompletableFuture<BasicArtDescription> getArtDescription(ArtRecognition r
                     }
 
                     return new BasicArtDescription(artName, null, artSummary, artType, null, null, null, null);
-
                 });
+
+        // If the art type is architecture or monument, we use the OpenAI API to get the missing data (artist, year, city, country)
+
+        //return new BasicArtDescription(artName, null, artSummary, artType, null, null, null, null);
```

Comment:
does this need to be removed?

Reply:
The comment in line 62 is an indication, the one from line 64 can be removed indeed.